src/myfunctions.js

```diff
@@ -0,0 +1,100 @@
+import {array} from './index.js'; // eslint-disable-line
```

Comment:
- [ ] The new functionality should be on a new separate Javascript file, please separate your code, so it's clean and more organized